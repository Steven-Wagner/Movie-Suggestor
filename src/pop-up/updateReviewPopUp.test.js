import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import PopUp from './pop-up';

describe('<PopUp', () => {
    it('renders without crashing', () => {
    const div = document.createElement('div');

    ReactDOM.render(
        <Router>
            <PopUp message= "Test Message"/>
        </Router>,
        div);
    ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the UI as expected', () => {
        const tree = renderer
        .create(
            <Router>
                <PopUp message= "Test Message"/>
            </Router>)
        .toJSON();
        expect(tree).toMatchSnapshot();  
    });
});