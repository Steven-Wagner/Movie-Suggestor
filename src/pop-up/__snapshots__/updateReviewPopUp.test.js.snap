// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PopUp renders the UI as expected 1`] = `
<div
  className="pop-up"
>
  <p>
    Test Message
  </p>
</div>
`;
